### ISSUE TITLE
setting tile missing sumerise

### ISSUE THREAD

**Issue by dimuthu3** [wasdennnoch/AndroidN-ify#635]
i have seen few tiles are not sumerised so can you add it.i have give sum ex for your ref.For some tiles i have no proper idea so need to look with android n settings but unfortunately android 7.0 beta device not in my hand now can you make it

![screenshot_20160727-121547](https://cloud.githubusercontent.com/assets/9978447/17172709/1f16c82a-5414-11e6-9dec-de83abcfe305.png)
![screenshot_20160727-122624](https://cloud.githubusercontent.com/assets/9978447/17172710/1f17b0be-5414-11e6-8c1e-3a513e1b3793.png)
![screenshot_20160727-122633 1](https://cloud.githubusercontent.com/assets/9978447/17172711/1f1aad50-5414-11e6-8ab6-5f453207216f.png)

**Comment by wasdennnoch** [comment]
The problem is that different ROMs add different entries, for example most entries in the first and second screenshot don't exist in vanilla Android which makes it harder to implement. Some tiles also don't have a summary like the Google tile, the Privacy one, Accessibility etc.
In the first screenshot only the notifications tile will get a summary and maybe the themes one, in the next one the Accounts tile should display the current account (maybe the profiles one), in the third screenshot the backup and printing tiles will get a summary. All other tiles probably not. I mean, I know if I have hw or sw keys, that's not useful.

**Comment by dimuthu3** [comment]
Ya you are right will wait for additional

**Comment by Madkita** [comment]
can you summarize according to this
security : current lockscreen security method like pattern,pin etc
home : current launcher
profiles : current profile
backup : on/off
privacy : privacy guard status on /off
accounts : auto sync on/off

**Comment by wasdennnoch** [comment]
I will put the summaries how they are in N.

**Comment by Madkita** [comment]
yeah but with all tiles summarized I will look better
